src/components/Shared/Icons/Buttons/PinButton/index.tsx

```diff
@@ -0,0 +1,16 @@
+import { IconButton } from '@components/Shared/Icons/Buttons/IconButton';
+import { useCallback } from 'react';
+
+export const PinButton = () => {
+  const handlePinClick = useCallback((e: React.SyntheticEvent) => {
+    e.preventDefault();
+  }, []);
+  return (
+    <IconButton
+      name="Pin"
+      tooltipLabel="Pinned Messages"
+      tooltipPlacement="bottom"
+      onClick={handlePinClick}
```

Comment:
```suggestion
      onClick={(e) => e.preventDefault()}
```

 and remove handlePinClick function above

Reply:
Resolved